src/com/dabomstew/pkrandom/gui/Bundle.properties

```diff
@@ -16,7 +16,7 @@ RandomizerGUI.wildPokemonARulePanel.border.title=Additional Rule
 RandomizerGUI.wpARNoneRB.toolTipText=Don't apply any other rules.
 RandomizerGUI.wpARNoneRB.text=None
 RandomizerGUI.spRandom2EvosRB.toolTipText=<html>Picks 3 random starter Pokemon to be used.<br />These Pokemon must have 2 evolution stages, e.g. be like the starters in the real games.
-RandomizerGUI.spRandom2EvosRB.text=Random (basic Pokemon with 2 evolutions)
+RandomizerGUI.spRandom2EvosRB.text=Random (Basic Pokemon with 2 evolutions)
```

Comment:
sure this should be capitalized?

Reply:
It's just a small formatting thing. It doesn't matter if it is or isn't capitalized for the GUI, but I think it looks better and fits better style-wise with a capital letter there.

Reply:
makes sense